pages/linux/resolvectl.md

```diff
@@ -0,0 +1,33 @@
+# resolvectl
+
+> Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services.
+> Introspect and reconfigure the DNS resolver.
```

Comment:
I don't think introspect is the correct word here.

Reply:
From  the man page:
"Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services; introspect and reconfigure the DNS resolver."

Reply:
I believe it does support  [network introspection commands](https://www.ibm.com/docs/en/api-connect/5.0.x?topic=interface-network-introspection-commands), though I admit I didn't include introspection in the examples.